projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.e2e-spec.ts

```diff
@@ -420,100 +419,112 @@ context('CPQ Configuration', () => {
 
   describe('Configuration Process', () => {
     it('should be able to add a configuration directly to the cart, navigate from the cart back to the configuration and update it, checkout and order', () => {
+      configuration.defineDeliveryModeAlias();
       configuration.goToPDPage(POWERTOOLS, PROD_CODE_CAM);
       configuration.clickOnAddToCartBtnOnPD();
       configuration.clickOnViewCartBtnOnPD();
 
-      type ovBundleInfo = {
-        name: string;
-        price?: string;
-        quantity?: string;
-      };
-      const ovBundleInfos: ovBundleInfo[] = [
-        {
-          name: 'SanDisk Extreme Pro 128GB SDXC',
-          price: '$100.00',
-          quantity: '1',
-        },
-        {
-          name: 'Canon RF 24-105mm f4L IS USM',
-          price: '$1,500.00',
-          quantity: '1',
-        },
-        {
-          name: 'LowePro Streetline SL 140',
-          price: '$110.00',
-          quantity: '1',
-        },
-      ];
-      configuration.checkAmountOfBundleItems(0, 3);
-
-      ovBundleInfos.forEach((line, bundleItemIndex) => {
-        configuration.checkBundleItemName(0, bundleItemIndex, line.name);
-        configuration.checkBundleItemPrice(0, bundleItemIndex, line.price);
-        configuration.checkBundleItemQuantity(
-          0,
-          bundleItemIndex,
-          line.quantity
-        );
+      cy.get('cx-mini-cart .count').then((elem) => {
+        const numberOfCartItems = Number(elem.text());
+        cy.log('numberOfCartItems = ' + numberOfCartItems);
+        editConfigurationFromCartEntry(numberOfCartItems);
+        checkout();
+        orderHistory();
       });
+    });
+  });
 
-      //We assume the last product in the cart is the one we added
-      configuration.clickOnEditConfigurationLink(
-        configuration.getNumberOfCartItems() - 1
+  function editConfigurationFromCartEntry(numberOfCartItems: number) {
+    //We assume the last product in the cart is the one we added
+    const cartEntryIndex: number = numberOfCartItems - 1;
+    type ovBundleInfo = {
+      name: string;
+      price?: string;
+      quantity?: string;
+    };
+    const ovBundleInfos: ovBundleInfo[] = [
+      {
+        name: 'SanDisk Extreme Pro 128GB SDXC',
+        price: '$100.00',
+        quantity: '1',
+      },
+      {
+        name: 'Canon RF 24-105mm f4L IS USM',
+        price: '$1,500.00',
+        quantity: '1',
+      },
+      {
+        name: 'LowePro Streetline SL 140',
+        price: '$110.00',
+        quantity: '1',
+      },
+    ];
+    configuration.checkAmountOfBundleItems(cartEntryIndex, 3);
```

Comment:
Could `configuration.checkAmountOfBundleItems(cartEntryIndex, ovBundleInfos.length)` be used instead?

Reply:
done